### PR TITLE
Still not right

### PR DESCRIPTION
**Pull request by chschulsie**
- When AKV is behind a firewall, it applies to both user-assigned AND system-assigned managed identity (which was the issue we had). Still not reflected correctly in the documentation. 
- The next sense also does not make sense: Once this option is enabled, available keys can't be listed in the SQL server TDE menu in the Azure portal. It should be either:  -- if this option is disabled, available keys can't be listed in the SQL server TDE menu in the Azure portal. or
-- This option must be enabled for keys to be successfully listed in the SQL server TDE menu in the Azure portal.

### CODE REVIEW

**Comment by Court72** [comment]
@GithubMirek

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: azure-sql/database/transparent-data-encryption-byok-identity.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ In addition to the system-assigned managed identity that is already supported fo
 
 ## Limitations and known issues
 
-- If the key vault is behind a VNet that uses a firewall, the option to **Allow Trusted Microsoft Services to bypass this firewall** must be enabled in the key vault's **Networking** menu if you want to use a user-assigned managed identity. Once this option is enabled, available keys can't be listed in the SQL server TDE menu in the Azure portal. To set an individual CMK, a *key identifier* must be used. When the option to **Allow Trusted Microsoft Services to bypass this firewall** isn't enabled, the following error is returned:
+- If the key vault is behind a VNet that uses a firewall, the option to **Allow Trusted Microsoft Services to bypass this firewall** must be enabled in the key vault's **Networking** menu if you want to use a user-assigned managed identity or system-assigned managed identity. If this option is disabled, available keys can't be listed in the SQL server TDE menu in the Azure portal. To set an individual CMK, a *key identifier* must be used. When the option to **Allow Trusted Microsoft Services to bypass this firewall** isn't enabled, the following error is returned:
```
Comment (strehan1993) [inline azure-sql/database/transparent-data-encryption-byok-identity.md]: **'Once this option is enabled'**
is correct, once AKV is behind a firewall it's not possible to view the available keys.

**Review by strehan1993 — CHANGES_REQUESTED** [review]
**'Once this option is enabled'**
is correct, once AKV is behind a firewall it's not possible to view the available keys.

Please scope this change to **'or system-assigned managed identity'**

File: azure-sql/database/transparent-data-encryption-byok-identity.md (hunk 1 above)
Comment (VanMSFT) [reply azure-sql/database/transparent-data-encryption-byok-identity.md]: I reverted the incorrect statement on this.

**Review by strehan1993 — APPROVED** [review]

**Comment by VanMSFT** [comment]
#sign-off

**Pull request merged**